stream-bot/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "stream-bot",
```

Comment:
why not use the  package.json that's used for the entire app? though this can work too. you just need to hook it up with the npm script for the main app to install these dependencies and run the bot's app.js